.github/workflows/build.yml

```diff
@@ -554,6 +554,8 @@ jobs:
         with:
           node-version: 20
       - uses: actions/setup-python@v5
+        with:
+          python-version: '3.8.10'
```

Comment:
Hmm, it complains here, but I guess it would make sense to pin _some_ version here? Thoughs on this (and which version that should be 😅 ) @JonasBa

Reply:
@mydea we can try LTS, but afaik devutils were removed in a minor python version, breaking node-gyp so we may need to reinstall some extra dependencies to be able to build the binaries.

Reply:
Did a quick search, [this](https://github.com/nodejs/node-gyp/issues/2869) is the issue I was talking about... Seems like 3.11 would be fine, so lets go with that

Reply:
The issue that you are quoting is **_closed_** because it was by vendoring in `packaging`.
https://github.com/nodejs/node-gyp/issues/2869#issuecomment-1769572922